dm/subtask_master.go

```diff
@@ -0,0 +1,175 @@
+package dm
+
+import (
+	"context"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tiflow/dm/dm/config"
+	"github.com/pingcap/tiflow/dm/dm/unit"
+	"github.com/pingcap/tiflow/dm/dumpling"
+	"github.com/pingcap/tiflow/dm/loader"
+	"github.com/pingcap/tiflow/dm/pkg/log"
+	"github.com/pingcap/tiflow/dm/syncer"
+	"go.uber.org/zap"
+
+	"github.com/hanfei1991/microcosm/lib"
+	"github.com/hanfei1991/microcosm/pkg/p2p"
+)
+
+var _ lib.JobMasterImpl = &SubTaskMaster{}
+
+// TODO: this is not the final jobmaster! I just want to make a runnable test.
+type SubTaskMaster struct {
+	lib.BaseJobMaster
+
+	cfg       *config.SubTaskConfig
+	workerSeq []lib.WorkerType
```

Comment:
Can you add a comment briefly explain the meaning of `workerSeq`?

Reply:
https://github.com/hanfei1991/microcosm/pull/114/commits/0f84f4205da07c2ff6f4f61d601b6e6bbfa4d151